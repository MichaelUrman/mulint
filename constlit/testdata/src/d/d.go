// Package d tests exceptions such as 1*time.Second
package d

import "time"

var T time.Duration = 1 // want "Replace `1` with `time.Nanosecond`"
var U time.Duration = 1 * time.Second
var V = time.Second * 1
var W = 3*time.Second + 1 // want "Replace `1` with `time.Nanosecond`"

var A int = 2 // OK; don't use time.Wed
