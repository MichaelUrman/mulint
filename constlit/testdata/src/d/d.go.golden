// Package d tests exceptions such as 1*time.Second
package d

import "time"

var T time.Duration = time.Nanosecond // want "Replace `1` with `time.Nanosecond`"
var U time.Duration = 1 * time.Second
var V = time.Second * 1
var W = 3*time.Second + time.Nanosecond // want "Replace `1` with `time.Nanosecond`"
